services/proxy/pkg/user/backend/cs3.go

```diff
@@ -85,6 +87,9 @@ func (c *cs3backend) GetUserByClaims(ctx context.Context, claim, value string, w
 				// https://github.com/owncloud/ocis/v2/issues/1825 for more context.
 				if user.Id.Type == cs3.UserType_USER_TYPE_PRIMARY {
 					c.logger.Info().Str("userid", user.Id.OpaqueId).Msg("user has no role assigned, assigning default user role")
+					// Updating context to have the Account-ID field and suffixing with _init
```

Comment:
Account-ID is required so that this call: [services/settings/pkg/service/v0/service.go # 389](https://github.com/owncloud/ocis/blob/master/services/settings/pkg/service/v0/service.go#L389) is able to retrieve it
And suffixing with "_init" _(or anything random)_ is required so that this call: [services/settings/pkg/service/v0/service.go # 394](https://github.com/owncloud/ocis/blob/master/services/settings/pkg/service/v0/service.go#L394) is passed because the `uuid` in context won't match the one in the request